### PR TITLE
TestFrustumOBB for model do not skip near_far

### LINKED ISSUE #7952: Model frustum culling ignore near/far z
**Describe the bug (REQUIRED)**

I find that frustum culling not worked for model in my game.
Looks like model ignore near/far z value

Model
https://github.com/defold/defold/blob/8f40197b0cf36995ac4b664db0b73943417c437d/engine/gamesys/src/gamesys/components/comp_model.cpp#L849
Mesh
https://github.com/defold/defold/blob/8f40197b0cf36995ac4b664db0b73943417c437d/engine/gamesys/src/gamesys/components/comp_mesh.cpp#L833

https://github.com/defold/defold/blob/8f40197b0cf36995ac4b664db0b73943417c437d/engine/dlib/src/dmsdk/dlib/intersection.h#L135

**To Reproduce (REQUIRED)**
1. Add model to scene
2. Set small far z value
3. You see model

**Expected behavior (REQUIRED)**
Model use near/far plan.

**Defold version (REQUIRED):**
  - Version 1.5.0

**Platforms (REQUIRED):**
 - Platforms: All

### PR DESCRIPTION
**Pull request by d954mas**
Fixed model frustum culling skipped near far plane

Fixes https://github.com/defold/defold/issues/7952
## PR checklist

* [ ] Code
	* [ ] Add engine and/or editor unit tests.
	* [ ] New and changed code follows the overall code style of existing code
	* [ ] Add comments where needed
* [ ] Documentation
	* [ ] Make sure that API documentation is updated in code comments
	* [ ] Make sure that manuals are updated (in github.com/defold/doc)
* [ ] Prepare pull request and affected issue for automatic release notes generator
	* [ ] Pull request - Write a message that explains what this pull request does. What was the problem? How was it solved? What are the changes to APIs or the new APIs introduced? This message will be used in the generated release notes. Make sure it is well written and understandable for a user of Defold.
	* [ ] Pull request - Write a pull request title that in a sentence summarises what the pull request does. Do not include "Issue-1234 ..." in the title. This text will be used in the generated release notes.
	* [ ] Pull request - Link the pull request to the issue(s) it is closing. Use on of the [approved closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
	* [ ] Affected issue - Assign the issue to a project. Do not assign the pull request to a project if there is an issue which the pull request closes.
	* [ ] Affected issue - Assign the "breaking change" label to the issue if introducing a breaking change.
	* [ ] Affected issue - Assign the "skip release notes" is the issue should not be included in the generated release notes.

### CODE REVIEW

**Review by JCash — CHANGES_REQUESTED** [review]

File: engine/gamesys/src/gamesys/components/comp_model.cpp [hunk 1]
```diff
@@ -846,7 +846,7 @@ namespace dmGameSystem
             dmRender::RenderListEntry* entry = &params.m_Entries[i];
             MeshRenderItem* render_item = (MeshRenderItem*)entry->m_UserData;
 
-            bool intersect = dmIntersection::TestFrustumOBB(frustum, render_item->m_World, render_item->m_AabbMin, render_item->m_AabbMax, true);
+            bool intersect = dmIntersection::TestFrustumOBB(frustum, render_item->m_World, render_item->m_AabbMin, render_item->m_AabbMax, false);
```
[the remark this answers is not in the document]
Comment (britzl) [reply engine/gamesys/src/gamesys/components/comp_model.cpp:849]: From a render script point of view something like this?

```lua
render.draw(model_pred, { frustum = frustum, skip_near_far = true })
```

[the remark this answers is not in the document]
Comment (d954mas) [reply engine/gamesys/src/gamesys/components/comp_model.cpp:849]: > This was a deliberate choice, as the number of extra culled objects are few in relation to the sides.

Thanks for answer. It is interesting that for meshes you use "false" but for model "true"

**Comment by d954mas** [comment]
I make a feature request  to add frustum_skip_near_far.
```lua
render.draw(model_pred, { frustum = frustum, frustum_skip_near_far = true })
```
https://github.com/defold/defold/issues/7959

**Comment by JCash** [comment]
Superceded by this PR: https://github.com/defold/defold/pull/7967

**Pull request closed without merging**